CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+<!-- Inherited from https://github.com/celo-org/celo-monorepo/blob/master/.github/CONTRIBUTING.md -->
+
+# Contributing
+
+Thank you for considering making a contribution to the Celo community!
+Everyone is encouraged to contribute, even the smallest fixes are welcome.
+
+If you'd like to contribute to Celo, please fork, fix, commit and send a
+pull request for the maintainers to review.
+
+If you wish to submit more complex changes, please sync with a core developer first.
+This will help ensure those changes are in line with the general philosophy of the project
+and enable you to get some early feedback.
+
+See the [contributing guide](https://docs.celo.org/community/contributing) for details on how to participate.
```

Comment:
```suggestion

```

I propose we remove this line and link to `https://docs.celo.org/community/contributing`. The page doesn't seem particularly well maintained. Updating and maintaining it to be linked here doesn't seem like a good use of time.

<img width="1058" alt="image" src="https://github.com/celo-org/.github/assets/46296830/a237e4a4-556d-4154-ab65-6a4de99ed7ac">